catalog/transformer/catalog-transformer-preview/pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>catalog-transformer-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>1.4</version>
```

Comment:
Is there a property for this version in one of the parent poms

Reply:
No there isn't, should I create one?

Reply:
There wasn't but I added one